usaspending_api/etl/tests/conftest_spark.py

```diff
@@ -116,7 +116,7 @@ def s3_unittest_data_bucket(s3_unittest_data_bucket_setup_and_teardown):
 
 
 @fixture(scope="session")
-def spark() -> SparkSession:
+def spark(tmp_path_factory) -> SparkSession:
```

Comment:
Where is the `tmp_path_factory` being defined?

Reply:
It's a built-in test fixture available from pytest.